Configuration/EcalTB/python/reco_application_2006_calibrate_cfg.py

```diff
@@ -14,12 +14,12 @@
     fileNames = cms.untracked.vstring('file:/u1/meridian/data/h4/2006/h4b.00013247.A.0.0.root')
 )
 
-process.ecal2006TBRecHit = cms.EDProducer("EcalRecHitProducer",
+process.load("RecoLocalCalo.EcalRecProducers.ecalRecHit_cfi")
```

Comment:
I'm curious why you use `process.load` instead of
```
from RecoLocalCalo.EcalRecProducers.ecalRecHit_cfi import ecalRecHit as _mod
```
as in the past? I may have simply missed the reason.

Reply:
OK, I think it's because this file has `process` already defined, so it's keeping the same style.